chart/templates/post-delete-hook-ServiceAccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.postDelete.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.postDelete.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "rancher.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
Interesting, I hadn't thought of using the delete policy on random objects like this. Should work.